tools/postgres/schema.sql

```diff
@@ -849,10 +849,11 @@ CREATE INDEX ON webknossos.invites(tokenValue);
 CREATE INDEX ON webknossos.annotation_privateLinks(accessToken);
 CREATE INDEX ON webknossos.shortLinks(key);
 CREATE INDEX ON webknossos.credit_transactions(credit_state);
+CREATE INDEX ON webknossos.dataset_mags(COALESCE(realPath, path));
+CREATE INDEX ON webknossos.dataset_layer_attachments(path);
```

Comment:
I added these indices to hopefully speed up the query listing mags & attachments that should be scanned for used storage. The idea is that the row_number operation is now faster as there exists n index. Previously only a part of the whole table was analyzed as the inner query was filtered by organization id. but now each query does a full table aggregation and thus we could use some  speed-up imo.

Else we could have a materialized view on dataset_mags that maintains the row_numbers. But as the table's content changes so frequently, this is not an option I think.